book/chrome.md

```diff
@@ -544,6 +543,8 @@ class Browser:
         self.window.bind("<Button-1>", self.handle_click)
 ```
 
+Remove these lines from `Tab`'s constructor.
```

Comment:
We usually give explicit instruction like this when refactoring, felt like this was missing.

Reply:
Not critical IMO